src/GithubClientFactory.ts

```diff
@@ -0,0 +1,41 @@
+import {GithubClientConfigs} from './Config';
+import {Logger} from "./Logger";
+/* const { throttlingPlugin } = require("@octokit/plugin-throttling");
+const { retryPlugin } = require("@octokit/plugin-retry");
+
+ */
+const Octokit = require("@octokit/rest").plugin(require("@octokit/plugin-throttling")).plugin(require("@octokit/plugin-retry"));
+
+export class GithubClientFactory  {
+    octokitClient;
+    constructor(private readonly config: GithubClientConfigs, logger :Logger) {
+        this.octokitClient = new Octokit({
```

Comment:
Étant donné que la version avec les plugins a un nouveau nom:

```suggestion
        this.octokitClient = new MyOctokit({
```